docs/usage.md

```diff
@@ -27,8 +48,33 @@ calculates a simple ratio between two strings
 
 
 ### partial_ratio
+: r: r
+Calculates the [ratio](#ratio) of the optimal string alignment
+
+Parameters:
+
+- **s1**: *str*
+
+    First string to compare.
+
+- **s2**: *str*
+
+    Second string to compare.
+
+- **processor**: *(Union[bool, Callable])*, default `utils.default_process`
+
+    Optional callable that reformats the strings.  is used by default, which lowercases the strings and trims whitespace.
+
+- **score_cutoff**: *float*, default `0`, optional
+
+    Optional argument for a score threshold as a float between 0 and 100. For `ratio < score_cutoff`, 0 is returned instead.
```

Comment:
Since this is essentially the same for all fuzz.<functions> (all have s1, s2, processor and score_cutoff) Which is why it is later on references as Parameters same as ...
So maybe we should simply add this as a general description for all fuzz functions (in front of fuzz.ratio) and then reference to it in the other descriptions. (e.g. fuzz.ratio has a score_cutoff aswell, which is currently missing here).

One thing I just realised to be wrong in the current docstrings is the default for processor, which is utils.default_process for all functions, but fuzz.ratio and fuzz.partial_ratio (the current docstring of them says they use this default aswell). The reason for this difference in behaviour is that fuzzywuzzy does default to processing for all other functions (even though it does not allow providing a custom function to them), but does not allow it for ratio and partial_ratio.

One more thing here is that these parameters description will change between programming languages, so they should be in tabs for Python/C++ aswell

Reply:
But we allow default for ratio and partial_ratio. Right?

Reply:
Yes. For all but ratio and partial_ratio the default is `processor=utils.default_process`, while it is `processor=None` for ratio and partial_ratio. It is possible to pass it `processor=utils.default_process` aswell (or pass any other similar function), or `processor=True` to use utils.default_process aswell